robottelo/config/settings.py

```diff
@@ -263,6 +263,32 @@ def validate(self):
         return validation_errors
 
 
+class VlanNetworkSettings(FeatureSettings):
```

Comment:
Can you move this class definition to match the alphabetical order? All other config classes are following this.